packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-target-group.ts

```diff
@@ -57,6 +57,22 @@ export interface ApplicationTargetGroupProps extends BaseTargetGroupProps {
    */
   readonly stickinessCookieDuration?: Duration;
 
+  /**
+   * The app cookie name.
```

Comment:
Can you update this docstring to match the one in `application-listener.ts`?

Reply:
Done.